index.js

```diff
@@ -3,10 +3,15 @@
 
 const reduct = require('reduct')
 const App = require('./src/app')
+const plugin = require('./src/plugin')
 
 if (require.main === module) {
   const app = reduct()(App)
-  app.listen()
+  plugin.connect().then(function () {
```

Comment:
I don't really understand why this is being called. Is the idea to not start the app unless moneyd access is present? It can be useful at times to use the moneyd gui when only the admin api is around

Reply:
Yeah I wanted to make sure that moneyd access is present before the wallet can use payment handler API. I can revert it though!